packages/logger-plugin/src/logger.module.ts

```diff
@@ -1,16 +1,16 @@
-import { NgModule, ModuleWithProviders, InjectionToken } from '@angular/core';
+import { InjectionToken, ModuleWithProviders, NgModule } from '@angular/core';
 import { NGXS_PLUGINS } from '@ngxs/store';
-
-import { NgxsLoggerPluginOptions, NGXS_LOGGER_PLUGIN_OPTIONS } from './symbols';
 import { NgxsLoggerPlugin } from './logger.plugin';
+import { NgxsLoggerPluginOptions, NGXS_LOGGER_PLUGIN_OPTIONS } from './symbols';
 
 export const USER_OPTIONS = new InjectionToken('LOGGER_USER_OPTIONS');
 
 export function loggerOptionsFactory(options: NgxsLoggerPluginOptions) {
   const defaultLoggerOptions: NgxsLoggerPluginOptions = {
     logger: console,
     collapsed: false,
-    disabled: false
+    disabled: false,
+    filter: () => true
```

Comment:
I think it would be breaking change, I think it is not necessary

Reply:
This is not a breaking change, because the option is optional.